SG-123 fix quantity and logo (#6)
retail-red/shopware-6: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).

## [Unreleased]
### Changed

> - to new Shopgate logo

Reviewer: ```suggestion
- replaced retail.red logo with the new one from Shopgate
```

### Fixed

> - product quantity change was providing data as a string instead of the required integer

Reviewer: ```suggestion
- product quantity sent as a string to the API, number required
```
